pkg/skaffold/build/tag/sha256.go

```diff
@@ -38,10 +38,12 @@ func (c *ChecksumTagger) GenerateFullyQualifiedImageName(workingDir string, opts
 	if opts == nil {
 		return "", fmt.Errorf("Tag options not provided")
 	}
-	digestSplit := strings.Split(opts.Digest, ":")
-	if len(digestSplit) != 2 {
-		return "", fmt.Errorf("Digest wrong format: %s, expected sha256:<checksum>", digestSplit)
+
+	digest := opts.Digest
```

Comment:
nit: don't need this var

Reply:
its used below on line 44

Reply:
It is used but I forgot to use it the line just after. Should be good now